app/controllers/concerns/analysis_pages.rb

```diff
@@ -8,6 +8,11 @@ def find_analysis_page_of_class(school, analysis_class)
     end
   end
 
+  def find_advice_page_of_class(analysis_class)
+    alert_type = AlertType.where("lower(class_name) = ?", analysis_class.downcase).first
+    return alert_type
```

Comment:
i think this can be simplified?
```
  def find_advice_page_of_class(analysis_class)
    AlertType.where("lower(class_name) = ?", analysis_class.downcase).first
  end
```